docset/winserver2022-ps/storagereplica/New-SRGroup.md

```diff
@@ -28,10 +28,10 @@ The `New-SRGroup` cmdlet creates a replication group. A replication group contai
 data volumes and an associated log volume. A replication group is the container for replication.
 
 > [!NOTE]
-> The **Compression** parameter is only available in
-> [Azure Stack HCI version 22H2](https://azure.microsoft.com/updates/public-preview-azure-stack-hci-version-22h2/) or later,
-> and Windows Server Datacenter: Azure Edition beginning with the 2022-09 Cumulative Update for Microsoft
-> server operating system version 21H2 for x64-based Systems
+> The **EnableCompression** parameter is only available in
```

Comment:
note: my change for WS2025 SMB compression does not apply here for Azure Edition, as AE only runs in VMs in Azure/Azure Stack, there is no SMB Direct/RDMA possible for VMs.

It could apply to Azure Stack HCI though, where RDMA is possible between nodes or some other HCI cluster